src/RelayClient.ts

```diff
@@ -442,7 +438,7 @@ class RelayClient extends EnvelopingEventEmitter {
     );
 
     //FIXME we should implement the relay selection strategy
```

Comment:
is this fixme still valid?

Reply:
I think so, because we haven't implemented a real strategy here. We are only iterating the array of preferred relays.